generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management.component.html.ejs

```diff
@@ -116,8 +116,13 @@
                     const relationshipFieldNamePlural = relationships[idx].relationshipFieldNamePlural;
                     const otherEntityName = relationships[idx].otherEntityName;
                     const otherEntityStateName = relationships[idx].otherEntityStateName;
-                    const otherEntityField = relationships[idx].otherEntityField;
+                    let otherEntityField = relationships[idx].otherEntityField;
                     const otherEntityFieldCapitalized = relationships[idx].otherEntityFieldCapitalized;
+                    const idSuffix = otherEntityName === 'authority' ? 'name' : 'id';
+                    // fallback for authority, if user has forgotten to define name....authorities do not have an ID
```

Comment:
This fallback isn't necessary, or it should be done at the generator.

Reply:
moved to entity index